packages/apps/docs/src/components/Layout/components/TitleHeader/TitleHeader.tsx

```diff
@@ -28,7 +30,9 @@ export const TitleHeader: FC<IProps> = ({ title, subTitle, avatar }) => {
 
   useEffect(() => {
     if (router.isReady) {
-      setLoaderHeaderClass(classNames(headerWrapperClass, headerLoadedClass));
+      setLoaderHeaderClass(
+        classNames(globalClass, headerWrapperClass, headerLoadedClass),
```

Comment:
instead of setting `globalClass` here. Adding it part of `headerWrapperClass` in css file makes sense. because for some reason the router takes time to be ready then it would be a glitch in UI